core/src/index/knowhere/CMakeLists.txt

```diff
@@ -56,15 +56,17 @@ set(index_srcs
         knowhere/index/vector_index/FaissBaseIndex.cpp
         knowhere/index/vector_index/IndexBinaryIDMAP.cpp
         knowhere/index/vector_index/IndexBinaryIVF.cpp
-        knowhere/index/vector_index/IndexHNSW.cpp
         knowhere/index/vector_index/IndexIDMAP.cpp
         knowhere/index/vector_index/IndexIVF.cpp
         knowhere/index/vector_index/IndexIVFPQ.cpp
         knowhere/index/vector_index/IndexIVFSQ.cpp
-        knowhere/index/vector_index/IndexNSG.cpp
         knowhere/index/vector_index/IndexType.cpp
         knowhere/index/vector_index/VecIndexFactory.cpp
         knowhere/index/vector_index/IndexAnnoy.cpp
+        knowhere/index/non_materialized_index/OffsetBaseIndex.cpp
```

Comment:
what "non_materialized_index" means ? Is there a better name ?
And list all files under vector_index to "vector_index_srcs",
list all files under "non_materialized_index" to another "xxx_index_srcs"

Reply:
Changed to vector_offset_index